common/main/en_JP.xml

```diff
@@ -0,0 +1,916 @@
+						<era type="169">↑↑↑</era>
+						<era type="170">↑↑↑</era>
+						<era type="171">↑↑↑</era>
+						<era type="172">↑↑↑</era>
+						<era type="173">↑↑↑</era>
+						<era type="174">↑↑↑</era>
+						<era type="175">↑↑↑</era>
+						<era type="176">↑↑↑</era>
+						<era type="177">↑↑↑</era>
+						<era type="178">↑↑↑</era>
+						<era type="179">↑↑↑</era>
+						<era type="180">↑↑↑</era>
+						<era type="181">↑↑↑</era>
+						<era type="182">↑↑↑</era>
+						<era type="183">↑↑↑</era>
+						<era type="184">↑↑↑</era>
+						<era type="185">↑↑↑</era>
+						<era type="186">↑↑↑</era>
+						<era type="187">↑↑↑</era>
+						<era type="188">↑↑↑</era>
+						<era type="189">↑↑↑</era>
+						<era type="190">↑↑↑</era>
+						<era type="191">↑↑↑</era>
+						<era type="192">↑↑↑</era>
+						<era type="193">↑↑↑</era>
+						<era type="194">↑↑↑</era>
+						<era type="195">↑↑↑</era>
+						<era type="196">↑↑↑</era>
+						<era type="197">↑↑↑</era>
+						<era type="198">↑↑↑</era>
+						<era type="199">↑↑↑</era>
+						<era type="200">↑↑↑</era>
+						<era type="201">↑↑↑</era>
+						<era type="202">↑↑↑</era>
+						<era type="203">↑↑↑</era>
+						<era type="204">↑↑↑</era>
+						<era type="205">↑↑↑</era>
+						<era type="206">↑↑↑</era>
+						<era type="207">↑↑↑</era>
+						<era type="208">↑↑↑</era>
+						<era type="209">↑↑↑</era>
+						<era type="210">↑↑↑</era>
+						<era type="211">↑↑↑</era>
+						<era type="212">↑↑↑</era>
+						<era type="213">↑↑↑</era>
+						<era type="214">↑↑↑</era>
+						<era type="215">↑↑↑</era>
+						<era type="216">↑↑↑</era>
+						<era type="217">↑↑↑</era>
+						<era type="218">↑↑↑</era>
+						<era type="219">↑↑↑</era>
+						<era type="220">↑↑↑</era>
+						<era type="221">↑↑↑</era>
+						<era type="222">↑↑↑</era>
+						<era type="223">↑↑↑</era>
+						<era type="224">↑↑↑</era>
+						<era type="225">↑↑↑</era>
+						<era type="226">↑↑↑</era>
+						<era type="227">↑↑↑</era>
+						<era type="228">↑↑↑</era>
+						<era type="229">↑↑↑</era>
+						<era type="230">↑↑↑</era>
+						<era type="231">↑↑↑</era>
+						<era type="232">↑↑↑</era>
+						<era type="233">↑↑↑</era>
+						<era type="234">↑↑↑</era>
+						<era type="235">↑↑↑</era>
+						<era type="236">↑↑↑</era>
+					</eraNarrow>
+				</eras>
+				<dateFormats>
+					<dateFormatLength type="full">		
+						<dateFormat>
+							<pattern>EEEE, MMMM d, y G</pattern>
+							<datetimeSkeleton>GyMMMMEEEEd</datetimeSkeleton>
+						</dateFormat>
+					</dateFormatLength>
+					<dateFormatLength type="long">
+						<dateFormat>
+							<pattern>MMMM d, y G</pattern>
+							<datetimeSkeleton>GyMMMMd</datetimeSkeleton>
+						</dateFormat>
+					</dateFormatLength>
+					<dateFormatLength type="medium">
+						<dateFormat>
+							<pattern>MMM d, y G</pattern>
+							<datetimeSkeleton>GyMMMd</datetimeSkeleton>
+						</dateFormat>
+					</dateFormatLength>
+					<dateFormatLength type="short">
+						<dateFormat>
+							<pattern>y/M/d GGGGG</pattern>
```

Comment:
Apple's en_JP locale has "GGGGG y/MM/dd" here (consistent with the MM/dd for gregorian)